core-web/package.json

```diff
@@ -1,259 +1,265 @@
-        "@tiptap/pm": "^2.1.13",
-        "@types/dragula": "^3.7.4",
-        "@types/googlemaps": "3.40.3",
-        "@types/jasmine": "4.0.3",
-        "@types/jasminewd2": "~2.0.8",
-        "@types/jest": "29.5.10",
-        "@types/lodash": "^4.14.202",
-        "@types/md5": "^2.3.5",
-        "@types/node": "18.16.9",
-        "@types/puppeteer": "^5.4.2",
-        "@types/react": "^18.2.46",
-        "@types/react-dom": "^18.2.18",
-        "@types/uuid": "^9.0.1",
-        "@types/webpack": "4.41.21",
-        "@typescript-eslint/eslint-plugin": "6.13.1",
-        "@typescript-eslint/parser": "6.13.1",
-        "babel-jest": "28.1.1",
-        "babel-loader": "^8.2.1",
-        "codelyzer": "^6.0.2",
-        "cypress": "^6.0.1",
-        "dotenv": "10.0.0",
-        "esbuild": "^0.17.5",
-        "eslint": "8.48.0",
-        "eslint-config-prettier": "9.0.0",
-        "eslint-plugin-ban": "^1.6.0",
-        "eslint-plugin-cypress": "2.15.1",
-        "eslint-plugin-import": "2.27.5",
-        "eslint-plugin-jsx-a11y": "6.7.1",
-        "eslint-plugin-react": "7.32.2",
-        "eslint-plugin-react-hooks": "4.6.0",
-        "eslint-plugin-storybook": "^0.6.15",
-        "flatpickr": "^4.5.7",
-        "gh-pages": "^3.1.0",
-        "html-webpack-plugin": "^5.5.0",
-        "http-proxy-middleware": "^2.0.6",
-        "husky": "^7.0.0",
-        "install": "^0.13.0",
-        "is-ci": "^3.0.1",
-        "jasmine-core": "4.2.0",
-        "jasmine-spec-reporter": "7.0.0",
-        "jest": "29.7.0",
-        "jest-cli": "29.7.0",
-        "jest-environment-jsdom": "29.7.0",
-        "jest-html-reporters": "^3.1.5",
-        "jest-junit": "^16.0.0",
-        "jest-preset-angular": "13.1.4",
-        "jsonc-eslint-parser": "^2.1.0",
-        "karma": "6.4.0",
-        "karma-chrome-launcher": "^3.1.0",
-        "karma-coverage": "~2.2.0",
-        "karma-coverage-istanbul-reporter": "~3.0.3",
-        "karma-jasmine": "5.1.0",
-        "karma-jasmine-html-reporter": "2.0.0",
-        "karma-junit-reporter": "^2.0.1",
-        "karma-spec-reporter": "^0.0.36",
-        "mock-socket": "^9.0.3",
-        "mockdate": "^3.0.5",
-        "monaco-editor": "^0.33.0",
-        "ng-mocks": "^14.12.1",
-        "npm": "^6.14.16",
-        "nx": "17.2.8",
-        "postcss": "^8.4.18",
-        "postcss-import": "14.1.0",
-        "postcss-preset-env": "7.5.0",
-        "postcss-url": "10.1.3",
-        "prettier": "^2.7.1",
-        "prettier-plugin-organize-attributes": "^0.0.5",
-        "pretty-quick": "^3.1.0",
-        "protractor": "~7.0.0",
-        "puppeteer": "~5.3.1",
-        "react-test-renderer": "^18.2.0",
-        "sass": "^1.56.2",
-        "ts-jest": "29.1.1",
-        "ts-node": "10.9.1",
-        "tslint": "~6.1.3",
-        "tslint-angular": "^3.0.3",
-        "typedoc": "^0.25.4",
-        "typescript": "~5.2.2",
-        "webpack": "^5.64.0",
-        "webpack-bundle-analyzer": "^4.5.0"
-    }
+  "name": "dotcms-ui",
```

Comment:
What happened here? looks like we have some lint or prettier issues.

Reply:
Fixed was the format:write at the moment of commit. Fixed added at the root properties to the prettier config file.